Library/Tracking/KSRAnalytics.swift

```diff
@@ -1126,6 +1143,20 @@ public final class KSRAnalytics {
 
   // MARK: - Search Events
 
+  /// Called when the search tab bar is clicked.
+  public func trackProjectSearchButtonClicked() {
+    let props = contextProperties(
+      ctaContext: .search,
+      locationContext: .globalNav
+    )
+
+    self.track(
+      event: NewApprovedEvent.ctaClicked.rawValue,
+      location: .search,
```

Comment:
Thinking we should rename this parameter, globally, to `page` to avoid confusion with the new `LocationContext` you added.

Reply:
Look at line 761 for reference

Reply:
Oh yeah, I thought about it too. I think I mentioned something similar last week or before, when I was working on `trackThanksPageViewed`, you said something about leaving it for now. I should have asked again.

I guess the new`LocationContext` I added requires that we change it.

Reply:
Updated.